flit/sdist.py

```diff
@@ -80,6 +80,7 @@ def find_nearest_pkg(rel_path):
 
 
 def include_path(p):
+    p = str(Path(p))
```

Comment:
> I'd like to use os.path.normpath() for this - I think it will produce the same result, but it makes it clearer why we're doing it.

✔️ That does feel more semantically clear.

> Maybe the normalisation should even be in the git wrapper interface?

✔️ I did make that the title of the PR, so it makes sense to make that the purpose of this PR.